### PR TITLE
9616: Feedback 

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js [hunk 1]
```diff
@@ -1,198 +1,160 @@
...
-
-    expect(caseDetail.docketNumber).toBeDefined();
-
-    cerebralTest.docketNumber = cerebralTest.leadDocketNumber =
-      caseDetail.docketNumber;
-    cerebralTest.consolidatedCasesThatShouldReceiveDocketEntries.push(
-      cerebralTest.docketNumber,
-    );
+  describe('Create a consolidated group', () => {
+    createConsolidatedGroup(cerebralTest);
```
Comment (ujahio) [inline web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js:20]: 👏🏿 👏🏿

File: web-client/integration-tests/docketClerkAddsAutomaticBlockCaseToTrial.test.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ describe('Adds automatic block case to trial', () => {
   };
 
   loginAs(cerebralTest, 'petitionsclerk@example.com');
-  petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
+  petitionsClerkCreatesNewCase(cerebralTest, true, { trialLocation });
```
Comment (mrinsin) [inline web-client/integration-tests/docketClerkAddsAutomaticBlockCaseToTrial.test.js]: Nit: Could the second argument be a named one? Hard to tell what `true` is being passed in for and I almost thought it was a typo when removing `fakeFile` :D

File: web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js [hunk 3]
```diff
@@ -1,198 +1,160 @@
...
-  it('login as a petitioner and create a case to consolidate with', async () => {
-    const caseDetail = await uploadPetition(cerebralTest);
-
-    expect(caseDetail.docketNumber).toBeDefined();
-
-    cerebralTest.docketNumber = caseDetail.docketNumber;
-    cerebralTest.consolidatedCasesThatShouldReceiveDocketEntries.push(
-      cerebralTest.docketNumber,
+  describe('Create a paper filing, serve and multi-docket', () => {
+    docketClerkAddsPaperFiledMultiDocketableDocketEntryAndServes(
```
Comment (mrinsin) [inline web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js:24]: These test refactorings are GREAT!

File: web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js [hunk 4]
```diff
@@ -1,198 +1,160 @@
...
-  });
+      const outboxQueue = cerebralTest.getState('workQueue');
 
-  it('verify a completed work item exists for each case in the consolidated group that the document was filed on from the document viewer', async () => {
-    await cerebralTest.runSequence('gotoWorkQueueSequence');
-    await cerebralTest.runSequence('chooseWorkQueueSequence', {
-      box: 'outbox',
-      queue: 'my',
-    });
+      for (const docketNumber of cerebralTest.consolidatedCasesThatShouldReceiveDocketEntries) {
```
Comment (mrinsin) [inline web-client/integration-tests/docketClerkAddsPaperFilingOnLeadCase.test.js:60]: 🎉

File: web-client/integration-tests/docketClerkAddsAutomaticBlockCaseToTrial.test.js (hunk 2 above)
Comment (rachelschneiderman) [reply web-client/integration-tests/docketClerkAddsAutomaticBlockCaseToTrial.test.js]: Yes good feedback. I'll do that!

File: web-client/integration-tests/petitionsClerkCaseDeadlineFromOrderForAmendedPetition.test.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ import { docketClerkViewsAutoGeneratedCaseDeadline } from './journey/docketClerk
 import { fakeFile, loginAs, setupTest, waitForCondition } from './helpers';
 import { petitionsClerkCreatesNewCaseFromPaper } from './journey/petitionsClerkCreatesNewCaseFromPaper';
 import { petitionsClerkReviewsPaperCaseBeforeServing } from './journey/petitionsClerkReviewsPaperCaseBeforeServing';
-import { petitionsClerkServesPaperCaseToIRS } from './userFlows/petitionsClerkServesPaperCaseToIRS';
```
Comment (ujahio) [inline web-client/integration-tests/petitionsClerkCaseDeadlineFromOrderForAmendedPetition.test.js:14]: Good catch

File: web-client/src/presenter/computeds/confirmInitiateCourtIssuedFilingServiceModalHelper.js [hunk 6]
```diff
@@ -44,9 +44,13 @@ export const confirmInitiateCourtIssuedFilingServiceModalHelper = (
 
   let parties;
   if (showConsolidatedCasesForService) {
+    const { consolidatedCasesToMultiDocketOn } = get(state.modal.form);
```
Comment (mrinsin) [inline web-client/src/presenter/computeds/confirmInitiateCourtIssuedFilingServiceModalHelper.js:47]: Chef's kiss to this name

**Review by ujahio — APPROVED** [review]

File: web-client/integration-tests/journey/consolidation/createConsolidatedGroup.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
+import { docketClerkConsolidatesCases } from '../docketClerkConsolidatesCases';
+import { docketClerkOpensCaseConsolidateModal } from '../docketClerkOpensCaseConsolidateModal';
+import { docketClerkSearchesForCaseToConsolidateWith } from '../docketClerkSearchesForCaseToConsolidateWith';
+import { docketClerkUpdatesCaseStatusToReadyForTrial } from '../docketClerkUpdatesCaseStatusToReadyForTrial';
+import { loginAs, uploadPetition } from '../../helpers';
+import { petitionsClerkServesElectronicCaseToIrs } from '../petitionsClerkServesElectronicCaseToIrs';
+
+export const createConsolidatedGroup = (
```
Comment (mrinsin) [inline web-client/integration-tests/journey/consolidation/createConsolidatedGroup.js:8]: NICE!!

**Review by mrinsin — APPROVED** [review]
So much great refactoring! I left one tiny nit-comment (that you already responded to) but otherwise, LGTM 😄

**Pull request merged**